examples/private_generics/Move.toml

```diff
@@ -7,9 +7,9 @@ version = "0.0.1"
 MoveosStdlib = { local = "../../moveos/moveos-stdlib/moveos-stdlib" }
 
 [addresses]
-rooch_examples = "_"
+rooch_examples = "0x42"
```

Comment:
The example project uses placeholders, which is more convenient for testing

Reply:
Okay, fixed.